common/app/renderers/DotcomRenderingService.scala

```diff
@@ -190,7 +221,7 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
     }
 
     val json = DotcomRenderingDataModel.toJson(dataModel)
-    post(ws, json, Configuration.rendering.baseURL + "/Article", page.metadata.cacheTime)
+    post(ws, json, Configuration.rendering.baseURL + path, page.metadata.cacheTime)
```

Comment:
I think something to consider is whether these requests will be proxied by MAPI. If they are, do we want caching on the Frontend side, or on the MAPI side, or both?